.phpstan/baseline.neon

```diff
@@ -3,3 +3,16 @@ parameters:
         - message: '#^Access to undefined constant static\(Liip\\ImagineBundle\\Config\\Filter\\Type\\FilterAbstract\)::NAME\.$#'
           paths:
               - ../src/Config/Filter/Type/FilterAbstract.php
+
+        - message: "#^Call to an undefined method Symfony\\\\Component\\\\Config\\\\Definition\\\\Builder\\\\NodeParentInterface\\:\\:validate\\(\\)\\.$#"
```

Comment:
could we get rid of this by providing a stub to phpstan? (i guess not, i suppose the class name would clash with the current symfony NodeParentInterface)

if not, can you please add a comment for which symfony version this is?

Reply:
I've added an `assert`

Reply:
nice!